doc/nrf/gs_installing.rst

```diff
@@ -385,6 +385,7 @@ To set up the toolchain, complete the following steps:
 #. Extract the contents of the root folder of the toolchain into a directory of your choice.
    The recommended folder is :file:`c:\gnuarmemb` on Windows and :file:`~/gnuarmemb` on Linux or macOS.
    Make sure that the folder name does not contain any spaces or special characters.
+   The contents are extracted to an additional folder whose name corresponds to the GNU Arm Embedded Toolchain version (*version-folder* in the following step).
```

Comment:
```suggestion
   The contents are extracted to another folder that corresponds to the GNU Arm Embedded Toolchain version (*version-folder* in the following step).
```

This is just a suggestion, not a required change. The original sentence felt a little bit long and complicated.